B/blis/build_tarballs.jl

```diff
@@ -0,0 +1,101 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "blis"
+version = v"0.7.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/flame/blis.git", "68b88aca6692c75a9f686187e6c4a4e196ae60a9")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd blis/
+
+for i in ./config/*/*.mk; do
+
+    # Building in container forbids -march options
+    sed -i "s/-march[^ ]*//g" $i
+    # Building in container forbids unsafe optimization.
+    sed -i "s/-ffast-math//g" $i
+    sed -i "s/-funsafe-math-optimizations//g" $i
+
+done
+
+case ${target} in
+
+    *"x86_64"*"linux"*) 
+        export BLI_CONFIG=x86_64
+        export BLI_THREAD=openmp
+        ;;
+    *"x86_64"*"w64"*) 
+        # MinGW doesn't support savexmm instructions
+        # Build only for AMD processors.
+        export BLI_CONFIG=amd64
+        export BLI_THREAD=openmp
+        ;;
+    *"x86_64"*"apple"*) 
+        export BLI_CONFIG=x86_64
+        export BLI_THREAD=openmp
+        export CC=gcc
+        export CXX=g++
+        ;;
+    *"x86_64"*"freebsd"*) 
+        export BLI_CONFIG=x86_64
+        export BLI_THREAD=openmp
+        export CC=gcc
+        export CXX=g++
+        ;;
+    *"aarch64"*"linux"*) 
+        # Aarch64 has no metaconfiguration support yet.
+        # Use Cortex-A57 for the moment.
+        export BLI_CONFIG=cortexa57
+        export BLI_THREAD=openmp
+        ;;
+    *"arm"*"linux"*) 
+        export BLI_CONFIG=cortexa9
+        export BLI_THREAD=none
+        ;;
+    *)
+        ;; 
+
+esac
+
+./configure -p ${prefix} -t ${BLI_THREAD} ${BLI_CONFIG}
+make -j${nproc}
+make install
+
+# Rename .dll for Windows targets.
+if [[ "${target}" == *"x86_64"*"w64"* ]]; then
+    mkdir ${prefix}/bin
+    mv ${prefix}/lib/libblis.3.dll ${prefix}/bin/libblis.dll
+fi
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux(:x86_64, libc=:musl),
+    Linux(:armv7l, libc=:glibc, call_abi=:eabihf),
+    Windows(:x86_64),
+    MacOS(:x86_64),
+    Linux(:x86_64, libc=:glibc),
+    Linux(:aarch64, libc=:glibc),
+    FreeBSD(:x86_64)
+]
+
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libblis", :blis)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = Dependency[
+]
```

Comment:
Note [the warning](https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=5794&view=logs&j=31e734e3-071a-557d-facd-25059fea1d24&t=6eae634a-8fe4-59d5-2d6a-4b4cf461852a&l=1402)
```
##[warning]libblis.so.3.0.0 contains a `cpuid` instruction; refusing to analyze for minimum instruction set, as it may dynamically select the proper instruction set internally.  Would have chosen skylake_avx512, instead choosing x86_64.
┌ Warning: To ensure that the correct version of libgomp is found at runtime, add the following entry to the list of dependencies of this builder
│ 
│     Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
└ @ BinaryBuilder.Auditor /depot/packages/BinaryBuilder/TCJai/src/auditor/compiler_abi.jl:4
```
```suggestion
dependencies = [
    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
]
```

Reply:
My fault... Thanks.